internal/handlers/hana/msg_compact.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hana
+
+import (
+	"context"
+
+	"github.com/FerretDB/FerretDB/internal/util/must"
+	"github.com/FerretDB/FerretDB/internal/wire"
+)
+
+// MsgCompact implements HandlerInterface.
```

Comment:
I think the comment is wrong. Isn't it be `Handler implements Interface`. Also can you please add `var _ Interface = (*Handler)(nil)` to provide compile-time check in order to make sure Handler implements the desired interface ?

Thanks

Reply:
> I think the comment is wrong. Isn't it be Handler implements Interface.

This method allows `*Handler` to implement `HandlerInterface`. This comment is also consistent with comments for all other methods.

> Also can you please add var _ Interface = (*Handler)(nil) to provide compile-time check in order to make sure Handler implements the desired interface ?

We already do that in `hana.go`.